scripts/migrations.js

```diff
@@ -0,0 +1,79 @@
+import Constants from '../constants.js';
+import Settings from '../settings.js';
+
+export default class Migrations {
+	constructor() {
+		this._settings = new Settings();
+	}
```

Comment:
I'm thinking that there should maybe be an `execute()` function that just executes all migrations in order to make this more extensible.

```js
async function execute() {
  await this._migrateCustomEffectsItemEffectDescriptions();
  // Additional migrations added later 
}
```

Reply:
Ideally, it would be nice to have some sort of array of migrations that execute based on what the last known migration was. You'd need to keep track of the global last migration, and have to do some comparisons between the last migration version and current version to know which migrations to execute. But this is likely a much bigger change.